tests/test_bqpipeline.py

```diff
@@ -63,24 +77,116 @@ def test_create_job_config_destination(self):
             self.assertEqual(cfg.destination.project, 'testproject')
 
     def test_create_job_config_flags(self):
-        bq = BQPipeline(
-            job_name='testjob', default_project='testproject', default_dataset='testdataset')
+        bq = bqpipeline.BQPipeline(
+            job_name='testjob', default_project='testproject',
+            default_dataset='testdataset')
         cfg = bq.create_job_config(batch=False, create=False, overwrite=False)
         self.assertEqual(cfg.priority, bigquery.QueryPriority.INTERACTIVE)
         self.assertEqual(cfg.create_disposition,
                          bigquery.job.CreateDisposition.CREATE_NEVER)
         self.assertEqual(cfg.write_disposition,
                          bigquery.job.WriteDisposition.WRITE_EMPTY)
 
+    def test_scalar_parameters(self):
+        result = bqpipeline.set_scalar_parameter('test', 'abc', typ='str')
+        self.assertEqual(result, bigquery.ScalarQueryParameter('test',
+                                                               'STRING', 'abc'))
+
+        result = bqpipeline.set_scalar_parameter('test', 1, typ='int')
+        self.assertEqual(result, bigquery.ScalarQueryParameter('test',
+                                                               'INT64', 1))
+
+        dt = datetime.datetime.now()
+        result = bqpipeline.set_scalar_parameter('test', dt, typ='datetime')
+        self.assertEqual(result, bigquery.ScalarQueryParameter('test',
+                                                               'TIMESTAMP', dt))
+
+        b = bytes(123)
```

Comment:
Tests don't cover [all BQ types supported as scalar parameters.](https://googleapis.github.io/google-cloud-python/latest/bigquery/generated/google.cloud.bigquery.query.ScalarQueryParameter.html#google.cloud.bigquery.query.ScalarQueryParameter)

Let's make sure we're covering float bool and date
```
‘STRING’, ‘INT64’, ‘FLOAT64’, ‘NUMERIC’, ‘BOOL’, ‘TIMESTAMP’, ‘DATETIME’, or ‘DATE’.
```

Reply:
Done.